src/supremm/preprocessors/Proc.py

```diff
@@ -1,42 +1,53 @@
 #!/usr/bin/env python
 """ Proc information pre-processor """
 
+import re
+import itertools
 from collections import Counter
 
 from supremm.plugin import PreProcessor
 from supremm.errors import ProcessingError
 from supremm.linuxhelpers import parsecpusallowed
-import re
-import itertools
 
-GROUP_RE = re.compile(r"cpuset:/slurm/uid_(\d+)/job_(\d+)/")
+SLURM_CGROUP_RE = re.compile(r"cpuset:/slurm/uid_(\d+)/job_(\d+)/")
+TORQUE_CGROUP_RE = re.compile(r"cpuset:/torque/(\d+(?:\[\d+\])?)(?:\.[^\.].*)?")
```

Comment:
Matches numeric job ID but doesn't seem to end up matching whole job ID which would be needed later to parse memory cgroups:

Dumped data from pcp archive:
```
inst [107427 or "107427 mpiexec"] value "hugetlb:/;memory:/torque/24.pitzer-batch.ten.osc.edu;pids:/system.slice/pbs_mom.service;blkio:/system.slice/pbs_mom.service;perf_event:/;net_prio,net_cls:/;cpuset:/torque/24.pitzer-batch.ten.osc.edu;cpuacct,cpu:/torque/24.pitzer-batch.ten.osc.edu;devices:/torque/24.pitzer-batch.ten.osc.edu;freezer:/;name=systemd:/system.slice/pbs_mom.service"
```

Groups:

```
>>> import re
>>> s = "hugetlb:/;memory:/torque/24.pitzer-batch.ten.osc.edu;pids:/system.slice/pbs_mom.service;blkio:/system.slice/pbs_mom.service;perf_event:/;net_prio,net_cls:/;cpuset:/torque/24.pitzer-batch.ten.osc.edu;cpuacct,cpu:/torque/24.pitzer-batch.ten.osc.edu;devices:/torque/24.pitzer-batch.ten.osc.edu;freezer:/;name=systemd:/system.slice/pbs_mom.service"
>>> re.compile(r"cpuset:/torque/((\d+(?:\[\d+\])?)(\.[^\;]+)?)").search(s).groups()
('24.pitzer-batch.ten.osc.edu', '24', '.pitzer-batch.ten.osc.edu')
```

Group 1 would be needed to access memory usage in cgroup plugins.  Group 2 would be retain current behavior of getting job ID.